learn/secrets-manager.mdx

```diff
@@ -73,6 +73,85 @@ Then, execute the `env` command to dump the environment variables of a session.
 hoop exec bash -i 'env' |grep PG_HOST
 ```
 
+## HashiCorp Vault Provider
+
+This provider expands environment variables from an [Vault Key Value Secrets Engine](https://developer.hashicorp.com/vault/docs/secrets/kv). It supports versions 1 and 2.
+
+### Credentials Configuration
+
+It requires the environment variables exported in the machine running the agent. The implemetation follows the specification of the [Vault Cli](https://developer.hashicorp.com/vault/docs/commands#environment-variables) and it's limited to the configuration below:
+
+| Name        | Required | Description |
+| ----------- | -------- | ----------- |
+| VAULT_ADDR  | yes | Address of the Vault server expressed as a URL and port, for example: `https://127.0.0.1:8200/`. |
+| VAULT_TOKEN | yes | Vault authentication token. Conceptually similar to a session token on a website, the VAULT_TOKEN environment variable holds the contents of the token |
+| VAULT_CACERT  | no | Path or inline base64 content of PEM-encoded CA certificate file on the local disk. This file is used to verify the Vault server's SSL certificate. |
+
+<Info>
+Example of how to define expose the env `VAULT_CACERT`
+
+```sh
+# laod from /tmp/ca.pem file
+export VAULT_CACERT=file:///tmp/ca.pem
+# load from inline base64 content
+export VAULT_CACERT=base64://<inline-base64-content>
+```
+</Info>
+
+### KV Version 1
+
+The command below will enable and configure a secret in a **KV version 1**
+
+```sh
+vault secrets enable -version=1 kv
+vault kv put kv/pgprod \
+    DBUSER=dbuser \
+    DBHOST=mydbhost.tld \
+    DBPASSWD=dbsupersecret \
+    DBPORT=5432
+```
+
+A connection could be mapped using the following syntax:
+
+- `_vaultkv1:SECRETNAME:SECRET-KEY `
+
+```sh
+hoop admin create conn pgprod --type database/postgres -a <agent-name> \
+    -e HOST=_vaultkv1:/kv/pgprod:DBHOST \
+    -e PORT=_vaultkv1:/kv/pgprod:DBPORT \
+    -e USER=_vaultkv1:/kv/pgprod:DBUSER \
+    -e PASS=_vaultkv1:/kv/pgprod:DBPASSWD
+```
```

Comment:
Instead of this guide using our CLI, it would be much better to have a print example with our webapp instead

Reply:
I got a nice screenshot for us to use here, I will share with you